db/fixtures/miq_product_features.yml

```diff
@@ -3971,6 +3971,10 @@
       :description: Edit Tags of Middleware Domains
       :feature_type: control
       :identifier: middleware_domain_tag
+    - :name: Stop Middleware Domain
+      :description: Stop Middleware Domain
+      :feature_type: admin
```

Comment:
@dclarizio does this mean that we should change review our MW features? Similar MW operations have `admin` on its feature type. Have a look: https://github.com/ManageIQ/manageiq/blob/a139ce6f4671660ef0f1fa6b5780ad221beca9d2/db/fixtures/miq_product_features.yml#L3785